chain/chain/src/store.rs

```diff
@@ -1917,10 +1907,11 @@ impl<'a> ChainStoreUpdate<'a> {
                             tries
                                 .revert_insertions(&trie_changes, shard_id, &mut store_update)
                                 .map(|_| {
-                                    store_update.delete(
+                                    self.gc_col(
                                         ColTrieChanges,
                                         &get_block_shard_id(&block_hash, shard_id),
-                                    );
+                                    )
+                                    .unwrap();
```

Comment:
Should we do "?" instead?

Reply:
Unused after voiding resulting type of `gc_col` execution.

Reply:
Yes please avoid panicking when it is not absolutely necessary.

Reply:
gc_col method doesn't return any error now, after gc_col_block_per_height is move to separate, so I'll make it return ()